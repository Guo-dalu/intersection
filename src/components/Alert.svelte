<script lang="ts">
	import { Alert } from 'flowbite-svelte'
	export let hasAlert = false
</script>

{#if hasAlert}
	<Alert dismissable class="absolute top-3 left-1/2 -translate-x-1/2">Error Happened!</Alert>
{/if}
