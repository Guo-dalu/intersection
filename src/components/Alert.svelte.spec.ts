import { expect, describe, it } from 'vitest'
import { render } from '@testing-library/svelte'
import Alert from './Alert.svelte'

describe('Alert Component', () => {
	it('should render correctly', () => {
		const { container } = render(Alert)

		expect(container).toMatchSnapshot()
	})
})
