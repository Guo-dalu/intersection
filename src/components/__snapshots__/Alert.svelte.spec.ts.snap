// Vitest Snapshot v1

exports[`Alert Component > should render correctly 1`] = `
<body>
  <div>
    
    <!--&lt;Alert&gt;-->
  </div>
</body>
`;
